src/MOI_wrapper.jl

```diff
@@ -2512,6 +2512,8 @@ struct ConstraintAttribute <: MOI.AbstractConstraintAttribute
     name::String
 end
 
+MOI.supports(::Optimizer, attr::ConstraintAttribute) = attr.name ∈ CONSTR_ATTR_TYPE
```

Comment:
It should be `attr.name \in keys(CONTR_ATTR_TYPE)`

Reply:
You should also add `, ::MOI.ConstraintIndex` to the signature

Reply:
Did both in https://github.com/JuliaOpt/Gurobi.jl/pull/243/commits/a6020f9f13aa811c48eb3681259d9986f2f46237

Reply:
Note: I actually added `, ::Type{<:MOI.ConstraintIndex}`, following similar examples in source code. This is consistent with the review at https://github.com/JuliaOpt/Gurobi.jl/pull/243#discussion_r319058347  using `typeof(ci)` in the call to `MOI.supports`.

Reply:
Indeed, `, ::Type{<:MOI.ConstraintIndex}` is what you needed to write